app/models/guide_article.ts

```diff
@@ -17,7 +22,15 @@ import { handleSortQuery } from "#scopes/sort_helper";
 
 import FileEntry from "./file_entry.js";
 
-export default class GuideArticle extends BaseModel {
+@MorphMap("guide_articles")
+export default class GuideArticle
+  extends BaseModel
+  implements AclModelInterface
+{
+  getModelId(): ModelIdType {
+    return this.id;
+  }
+
```

Comment:
solvronis: digs deep into adonis internals to figure out which column is the primary key
this library: yeah whatever, just give use a method, we're too lazy